src/herder/Herder.h

```diff
@@ -121,6 +121,9 @@ class Herder
     // and local state
     virtual uint32_t getCurrentLedgerSeq() const = 0;
 
+    // return the smallest ledger number we need messages for when asking peers
+    virtual uint32 getLowNeedsSCPLedgerSeq() const = 0;
```

Comment:
The name is a bit confusing, how about something like `getMinLedgerSeqToAskPeers`?

Reply:
done